src/components/home/Main.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { ProfileCard } from '@/components/home/ProfileCard'
+import classes from '@/components/home/style.css'
+import useGetUsers from '@/components/home/useGetUsers'
+
+export const Main = () => {
+  const { Users, transferToProfile } = useGetUsers()
+  const myId = '3f328652-f4bb-4254-972a-d70489794a25'
+
+  return (
+    <div className={classes.container}>
+      {Object.keys(Users)
+        .filter((key) => key !== myId)
+        .map((key, id) => {
+          return <ProfileCard key={id} User={Users[key]} transferToProfile={transferToProfile} />
```

Comment:
keyはstring型でもOKなはずなので、
```suggestion
          return <ProfileCard key={key} User={Users[key]} transferToProfile={transferToProfile} />
```
でも動作すると思います！

Reply:
たしかにそうですね、修正しました